README.md

```diff
@@ -10,7 +10,7 @@ Web of Things [Scripting API](https://w3c.github.io/wot-scripting-api/) modelled
 after the WoT reference implementation
 [node-wot](https://github.com/eclipse/thingweb.node-wot).
 At the moment, it supports interacting with Things using the Constrained Application
-Protocol (CoAP) and the Hypertext Transfer Protocol (HTTP).
+Protocol (CoAP), the Hypertext Transfer Protocol (HTTP), and the MQTT protocol.
```

Comment:
(As a matter of fact, MQTT seems to not stand for anything since 2013 (see [here](https://www.oasis-open.org/committees/download.php/49028/OASIS_MQTT_TC_minutes_25042013.pdf)), therefore I haven't written it out here.)